manifests/w/WinFsp/WinFsp/1.9/WinFsp.WinFsp.locale.en-US.yaml

```diff
@@ -0,0 +1,18 @@
+# Created using wingetcreate 0.3.0.6
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: WinFsp.WinFsp
+PackageVersion: 1.9
+PackageLocale: en-US
+Publisher: Navimatics LLC
+PackageName: WinFsp 2021

```

Comment:
```suggestion
PackageName: WinFsp
```

Reply:
Thought PackageName needed to match the Add and Remove Programs listing?
![image](https://user-images.githubusercontent.com/1053504/132876251-22a19739-f984-42e2-b9d7-518c59e811f7.png)

Reply:
Since the `PackageIdentifier` isn't called `WinFsp.WinFsp.2021` it's not needed and the version number in the `PackageName` is being saved for the 1.1 schema.